parquet-avro/src/main/java/org/apache/parquet/avro/AvroWriteSupport.java

```diff
@@ -219,14 +219,14 @@ private void writeValue(Type type, Schema avroSchema, Object value) {
     } else if (avroType.equals(Schema.Type.UNION)) {
       writeUnion((GroupType) type, nonNullAvroSchema, value);
     } else if (avroType.equals(Schema.Type.FIXED)) {
-      recordConsumer.addBinary(Binary.fromByteArray(((GenericFixed) value).bytes()));
```

Comment:
I don't think the intent is captured correctly here. Avro objects like `Fixed` and `Utf8` will be reused if reading from an Avro file and writing to Parquet. In that case, shouldn't these methods use `Binary.fromReusedByteArray` and similar?